apps/condo/domains/organization/integrations/sbbol/sync/syncSbbolSubscriptionPaymentRequestsState.js

```diff
@@ -36,14 +37,17 @@ const syncSbbolSubscriptionPaymentRequestStateFor = async (payment, fintechApi)
                 }
                 if (status === SUBSCRIPTION_PAYMENT_STATUS.DONE) {
                     const now = dayjs()
+                    const startAt = now.toISOString()
+                    const finishAt = now.add(1, 'year').toISOString()
                     await ServiceSubscription.create(context, {
                         ...dvSenderFields,
                         type: SUBSCRIPTION_TYPE.SBBOL,
                         isTrial: false,
                         organization: { connect: { id: payment.organization.id } },
-                        startAt: now.toISOString(),
-                        finishAt: now.add(1, 'year').toISOString(),
+                        startAt,
+                        finishAt,
                     })
+                    pushSubscriptionActivationToSalesCRM(data.payerInn, startAt, finishAt, false)
```

Comment:
Why here and not inside afterChange hook ?

Reply:
I think, we can use hooks.